src/js/build.sh

```diff
@@ -1,8 +1,24 @@
-#!/usr/bin/bash
+#!/bin/bash
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
 
+set -ex
 root=$(git rev-parse --show-toplevel)
-rm -rf "$root"/build/node/output/*
+js_root="${root}/src/js/"
+cd "$js_root"
+build_dir="${V_BUILD_DIR:-build}/node/output"
```

Comment:
Fuego change, don't forget to update the vtools PR to change the way `vtools build js` works. It should also pass `V_BUILD_DIR` as a new environment variable